pkg/util/util.go

```diff
@@ -225,3 +253,17 @@ func GetRandomString(length int) (string, error) {
 	}
 	return b.String(), nil
 }
+
+func HasExternalListeners(kafkaClusterSpec v1beta1.KafkaClusterSpec) bool {
+	// Has global External Listener
+	if kafkaClusterSpec.ListenersConfig.ExternalListeners != nil {
+		return true
+	}
+	// No global External Listener. All BrokerGroups must declare at least 1 ExternalListener
+	for _, brokerConfig := range kafkaClusterSpec.BrokerConfigGroups {
```

Comment:
brokerConfig is misleading when it is really a brokerGroupConfig.

Please change this in all other parts for clarity